### PR TITLE
Added google analytics and fixed newsletter form

### PR DESCRIPTION
**Pull request by patrikulus**
As initiative creators we wan't to know a little bit about our users.

## Description
Added google analytics script. Also fixed issue with newsletter form size.

## Where should the reviewer start?
Check if analytics data are increasing after loading page. It should work from localhost as well. Analytics account access needed. 

## How has this been tested?
Checked if site counter was increased by loading page from localhost.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by WTobor** [comment]
Do we need to create Privacy Policy? Are we using cookies? https://www.iubenda.com/en/help/11994-privacy-policy-for-google-analytics

**Comment by WTobor** [comment]
Here is a good example of creating Privacy Policy
post: https://csharp.christiannagel.com/2018/05/24/gdpr/
code: https://github.com/ProfessionalCSharp/MoreSamples/tree/master/ASPNETCore/GDPRSample/GDPRSample

**Comment by WTobor** [comment]
@patrikulus or @smiechuskce please look at cookies policy

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -71,7 +81,7 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js"
```
Comment (patrikulus) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: I hope you ensured that Bootstrap version we use supports jQuery 3 ;)

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: yes :)

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml]: http://blog.getbootstrap.com/2016/07/25/bootstrap-3-3-7-released/

**Pull request merged**